src/main/java/org/repodriller/RepositoryMining.java

```diff
@@ -42,18 +40,19 @@
 
 public class RepositoryMining {
 
-	private List<SCMRepository> repos;
-	private Map<CommitVisitor, PersistenceMechanism> visitors;
-	
+	private static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss");
```

Comment:
In this case an Util class could be acceptable. I'm trying to find some memory leak points of my study, and a `new SimpleDateFormat()` every commit causes a little more load to GC. I'm also writing explicit `final` modifier in every possible variable of my visitor and processors.